### PR TITLE
Edited 2 files in Container Deployment section. More to come.

### PR DESCRIPTION
**Pull request by philip-menzies**

### CODE REVIEW

File: docs/get_started/plan_deployment/container_deployment/application_architecture.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Content Composer [HCL Content Composer](../../../manage_content/wcm_authoring/co
 
 **Type:** Stateful
 
-Digital Asset Management requires the Ring API be deployed and operational to communicate with the DX Core, and Image Processor components to perform image manipulation, and Persistence to store its application data. See [Digital Asset Management](../../../manage_content/digital_assets/index.md)
+Digital Asset Management requires the Ring API be deployed and working to communicate with the DX Core, image processor components to perform image manipulation, and persistence to store its application data. See [Digital Asset Management](../../../manage_content/digital_assets/index.md)
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/application_architecture.md:48]:
```suggestion
Digital Asset Management requires the Ring API be deployed and working to communicate with the DX Core, image processor components to perform image manipulation, and persistence to store its application data. See [Digital Asset Management](../../../manage_content/digital_assets/index.md).
```

File: docs/get_started/plan_deployment/container_deployment/dam_persistence_architecture.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 # Digital Asset Management persistence
 
-This topic describes the components of the Digital Asset Management persistence. The updated DAM persistence feature is available from HCL Digital Experience 9.5 Container Update CF198 and later.
+This topic describes the components of the Digital Asset Management \(DAM\) persistence. The updated DAM persistence feature is available with HCL Digital Experience 9.5 Container Update CF198 and later.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/dam_persistence_architecture.md:3]:
```suggestion
This topic describes the components of the Digital Asset Management (DAM) persistence. The updated DAM persistence feature is available with HCL Digital Experience 9.5 Container Update CF198 and later.
```

File: docs/get_started/plan_deployment/container_deployment/index.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
-Helm is a tool for managing Kubernetes applications and deployments. It allows for packaging all required resource definitions into a single package, called a Helm Chart. The Helm Charts provide a convenient way to define application deployments with a predefined set of configurable items. Furthermore, Helm Charts are written using declarative definitions, applying `yaml` structures and go templates. This approach provides administrators with transparency about the operations the Helm Chart is performing during the DX 9.5 container deployment.
+Helm is a tool for managing Kubernetes applications and deployments. With Helm, helps you package all required resource definitions into a single package, called a Helm chart. The Helm charts provide a convenient way to define application deployments with a predefined set of configurable items. Furthermore, Helm charts are written with declarative definitions, applying `yaml` structures and `go` templates. This approach provides administrators with transparency about the operations the Helm chart performs during the DX 9.5 container deployment.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/index.md:11]:
```suggestion
Helm is a tool for managing Kubernetes applications and deployments. Helm helps you package all required resource definitions into a single package, called a Helm chart. The Helm charts provide a convenient way to define application deployments with a predefined set of configurable items. Furthermore, Helm charts are written with declarative definitions, applying `yaml` structures and `go` templates. This approach provides administrators with transparency about the operations the Helm chart performs during the DX 9.5 container deployment.
```

File: docs/get_started/plan_deployment/container_deployment/index.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 
-In addition to foundation packaging and installation capabilities, Helm can also be used to modify and upgrade existing deployments, if the Helm Charts are built to support this. Configuration changes and application upgrades can both be managed using Helm.
+In addition to foundation packaging and installation capabilities, you can also use Helm to modify and upgrade existing deployments, if the Helm charts are built to support these actions. Configuration changes and application upgrades can both be managed by using Helm.
 
-For more information about Helm, please reference documentation available on Helm topics for Red Hat [Red Hat OpenShift](https://docs.openshift.com/container-platform/4.7/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html) and [Kubernetes](https://helm.sh/) container platforms.
+For more information about Helm, refer documentation about Helm for Red Hat container platforms:  [Red Hat OpenShift](https://docs.openshift.com/container-platform/4.7/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html) and [Kubernetes](https://helm.sh/).
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/index.md:15]:
```suggestion
For more information about Helm, refer  to documentation about Helm for Red Hat container platforms:  [Red Hat OpenShift](https://docs.openshift.com/container-platform/4.7/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html) and [Kubernetes](https://helm.sh/).
```

File: docs/get_started/plan_deployment/container_deployment/index.md [hunk 5]
```diff
@@ -29,57 +29,57 @@ hcl-dx-deployment/
 
 ```
 
--   **templates**: The templates directory contains all resource definitions, for example, Services and Pods.
--   **values.yaml**: The values.yaml contains all default values for a deployment. It is possible to customize the deployment overwriting the default values of the values.yaml.
--   **values.schema.json**: To validate the values entered for a deployment, the values.schema.json provides configuration whenever an install or upgrade is performed with Helm.
+-   **templates**: The `templates` directory contains all resource definitions,, for example, services and pods.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/index.md:32]:
```suggestion
-   **templates**: The `templates` directory contains all resource definitions, for example, services and pods.
```

File: docs/get_started/plan_deployment/container_deployment/limitations_requirements.md [hunk 6]
```diff
@@ -1,62 +1,61 @@
...
+-   A `persistence-node` relies on PostgreSQL, which requires the use of hard links. Storage systems \(like Azure Files\) that do not support hard links cannot be used. For more information, see the [Microsoft documentation for features not supported by the Azure File service](https://docs.microsoft.com/en-us/rest/api/storageservices/features-not-supported-by-the-azure-file-service).
+-   All DX applications require the use of symbolic links and soft links. Storage systems must support the use of symbolic links and soft links. If you use Azure Files, you must enable `mountOptions` settings of the StorageClass by using `mfsymlinks`. For more information, see the [Microsoft documentation on troubleshooting Azure Files on Linux \(SMB\)](https://docs.microsoft.com/en-us/azure/storage/files/storage-troubleshoot-linux-file-connection-problems#cannot-create-symbolic-links---ln-failed-to-create-symbolic-link-t-operation-not-supported).
+-   You can configure volume sizes individually per volume and these dependend on the respective usage. For more information, see the following Help Center topics:
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/limitations_requirements.md:20]:
```suggestion
-   You can configure volume sizes individually per volume and these depend on the respective usage. For more information, see the following Help Center topics:
```

File: docs/get_started/plan_deployment/container_deployment/limitations_requirements.md [hunk 7]
```diff
@@ -88,23 +87,23 @@ To deploy HCL Digital Experience 9.5 CF200 to the supported Kubernetes platforms
...
 
 !!!note
-    Remember that the overall sums for CPU and memory include all components of HCL DX. 
-    In most cases, you will only want to deploy a subset of all components, therefore the minimal system requirements will decrease accordingly.
+    The overall sums for CPU and memory include all components of HCL DX. 
+    In most cases, you only want to deploy a subset of all components. Therefore, the minimal system requirements  decrease accordingly.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/limitations_requirements.md:94]:
```suggestion
    In most cases, you only want to deploy a subset of all components. Therefore, the minimal system requirements decrease accordingly.
```

File: docs/get_started/plan_deployment/container_deployment/persistent_volumes.md [hunk 8]
```diff
@@ -1,31 +1,29 @@
...
+You must use PVs because Kubernetes Pods do not have their own persistent file storage. For more information about PVs, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
 
-There are two types of PersistentVolumes used for DX: [`ReadWriteOnce (RWO)` and `ReadWriteMany (RWX)`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+You use two types of PersistentVolumes with DX: [`ReadWriteOnce (RWO)` and `ReadWriteMany (RWX)`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
 
 
 ## Core
 
-The DX 9.5 Core application requires multiple PersistentVolumes (PVs) of different types and sizes for its operation. From an application perspective, there are three different things that need to be persisted:
+The DX 9.5 Core application requires multiple PVs of different types and sizes to operate. From an application perspective, three items things that have to persist:
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/persistent_volumes.md:14]:
```suggestion
The DX 9.5 Core application requires multiple PVs of different types and sizes to operate. From an application perspective, there are three items that have to persist:
```

File: docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md [hunk 9]
```diff
@@ -3,52 +3,52 @@ id: DX_Rendering_Sizing_Small_Configuration_Results
...
+DX sizing is one of the goals of DX performance tests. DX sizing aims to identify the reliable Kubernetes environment for small, medium, and large DX configurations. This topic presents the important KPIs (for example, number of concurrent users, average response time, and throughput) in small configurations. A similar approach will be applied for medium and large configurations. This topic also discusses how adjustments to the pod configuration can make significant improvements in system responsiveness.
 
-This sizing work started with rendering scenarios of WCM, Portlets, and DAM with a rendering setup enabled in AWS/Native-Kube. The Apache JMeter tool was used for performance tests.
+This sizing work started with rendering scenarios of web content management (WCM), portlets, and digital asset management (DAM) with a rendering setup enabled in AWS/Native-Kubernetes. The Apache JMeter tool was used for performance tests.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md:12]:
```suggestion
This sizing work started with rendering scenarios of Web Content Management (WCM), portlets, and Digital Asset Management (DAM) with a rendering setup enabled in AWS/Native-Kubernetes. The Apache JMeter tool was used for performance tests.
```

File: docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md [hunk 10]
```diff
@@ -131,27 +131,27 @@ To see the small-config-values.yaml file in the hcl-dx-deployment helm chart, re
...
+- Half of the content items (spread evenly over each type described previously) are profiled with the keyword "MENU."
 
-- There are 20 test portal pages created under the label "PerformanceTest". Each has a friendly URL in the form "<context-root>/perf/page-xx".
+- There are 20 test-portal pages created under the "PerformanceTest" label. Each has a URL in the form "<context-root>/perf/page-xx".
 
-- Each page contains six WCM Viewer portlets that show content below one of the twenty top-level site areas. For example, Pages 01 to 04 show content from site areas "SA01" through "SA04" in library "PerformanceTestContent01", and pages 05 to 08 show content from site areas "SA01" through "SA04" in library "PerformanceTestContent02" etc.
+- Each page contains 6 WCM viewer portlets that show content below one of the 20 top-level site areas. For example, Pages 01 to 04 show content from site areas "SA01" through "SA04" in the "PerformanceTestContent01" library, and pages 05 to 08 show content from site areas "SA01" through "SA04" in the "PerformanceTestContent02"  library, and so on.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md:138]: Aren't we supposed to spell out numbers less than 10? 
```suggestion
- Each page contains six WCM viewer portlets that show content below one of the 20 top-level site areas. For example, Pages 01 to 04 show content from site areas "SA01" through "SA04" in the "PerformanceTestContent01" library, and pages 05 to 08 show content from site areas "SA01" through "SA04" in the "PerformanceTestContent02"  library, and so on.
```

File: docs/get_started/plan_deployment/container_deployment/persistent_volumes.md [hunk 11]
```diff
@@ -1,31 +1,29 @@
...
 -   Transaction logs (RWO)
 
-While the profile needs to be shared between all DX 9.5 Core Pods, the logs and transaction logs are per Pod and not shared. This means that the persistent volume used for the profile must be `ReadWriteMany` (RWX).
+Although the profile must be shared between all DX 9.5 Core pods, the logs and transaction logs are per pod and not shared. This shared status means that the persistent volume for the profile must be `ReadWriteMany` (RWX).
 
-The persistent volumes used for logs and transaction logs are established as `ReadWriteOnce` (RWO) for proper operations performance.
+The persistent volumes for logs and transaction logs are established as `ReadWriteOnce` (RWO) for correct operation performance.
 
-For example, in a DX 9.5 Core deployment:
+For example, in a DX 9.5 Core deployment, to deploy one Core pod, 3 PVs are required. To deploy two more Core pods, the number of required PVs increases by two, resulting in 5 required PVs. The additional pods share the existing profile PV with the first Pod, but require their own log and transaction log PVs.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/persistent_volumes.md:24]:
```suggestion
For example, in a DX 9.5 Core deployment, to deploy one Core pod, three PVs are required. To deploy two more Core pods, the number of required PVs increases by two, resulting in five required PVs. The additional pods share the existing profile PV with the first Pod, but require their own log and transaction log PVs.
```

File: docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md [hunk 12]
```diff
@@ -131,27 +131,27 @@ To see the small-config-values.yaml file in the hcl-dx-deployment helm chart, re
...
 - The final portlet on each page shows a menu of items and is scoped to the top-level site area. It also selects only those items profiled with the "MENU" keyword.
 
-- A total of 99999 users were added to openLDAP as authenticated users.
+- A total of 99,999 users were added to openLDAP as authenticated users.
 
-## DAM Default Test Data - 2,500 assets
+## DAM default test data: 2,500 assets
 
-- 2,500 assets were uploaded. These include images (136kb, .jpg), documents (199kb, .docx), and videos (1.1mb, .mp4) to preheat. After preheating with 2,500 assets, 15 assets were uploaded and rendered for one hour.
+- Testers uploaded 2,500 assets. These assets include images (136kb, .jpg), documents (199kb, .docx), and videos (1.1mb, .mp4) to preheat. After preloading 2,500 assets, 15 assets were uploaded and rendered for 1 hour.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md:150]:
```suggestion
- Testers uploaded 2,500 assets. These assets include images (136kb, .jpg), documents (199kb, .docx), and videos (1.1mb, .mp4) to preheat. After preloading 2,500 assets, 15 assets were uploaded and rendered for one hour.
```

File: docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md [hunk 13]
```diff
@@ -131,27 +131,27 @@ To see the small-config-values.yaml file in the hcl-dx-deployment helm chart, re
...
 
 - Details of 15 assets were uploaded for rendering.
       
-- There are 19 URLs in total, three of which are custom URLs, eight are uuid URLs, and eight are friendly URLs which are combinations of original, tablet, smartphone, and desktop renditions.
+- There are 19 URLs in total, three of which are custom URLs, 8 are uuid URLs, and 8 are short URLs, which are combinations of original, tablet, smartphone, and desktop renditions.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/rm_container/dx_performance_small_cfg.md:154]: Aren't we supposed to spell out numbers less than 10? (Microsoft Style Guide)

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md [hunk 14]
```diff
@@ -1,26 +1,26 @@
 # WCM Content AI Analysis
 
-This topic describes how to enable AI analysis for a WCM Content in Kubernetes Deployment. This also discusses steps on how to configure a content AI provider to be used for AI analysis. The AI analysis for a WCM Content feature is available from HCL Digital Experience 9.5 Container Update CF213 and higher.
+You can enable AI analysis for web content management (WCM) content in a Kubernetes deployment. You can also configure a content AI provider to be used for AI analysis. The AI analysis for a WCM content feature is available from HCL Digital Experience 9.5 Container Update CF213 and higher.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:3]:
```suggestion
You can enable AI analysis for Web Content Management (WCM) content in a Kubernetes deployment. You can also configure a content AI provider to be used for AI analysis. The AI analysis for a WCM content feature is available from HCL Digital Experience 9.5 Container Update CF213 and higher.
```

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md [hunk 15]
```diff
@@ -1,26 +1,26 @@
...
 
-## Content AI Provider Overview
+## Content AI provider overview
 
-### OpenAI ChatGPT Overview
+### OpenAI ChatGPT overview
 
-OpenAI is the AI research and deployment company behind ChatGPT. When you sign up with ChatGPT, it provides API access via an API key. After signing up at [https://platform.openai.com/playground](https://platform.openai.com/playground), you can create a personal account with limited access or a corporate account. The playground can be used to experiment with the API as well. A highlight of the API is that it accepts natural language commands similar to the ChatGPT chatbot. 
+OpenAI is the AI research and deployment company behind ChatGPT. When you sign up with ChatGPT, it provides API access through an API key. After signing up at [https://platform.openai.com/playground](https://platform.openai.com/playground), you can create a personal account with limited access or a corporate account. You can use the playground to experiment with the API too. A highlight of the API is that it accepts natural language commands similar to the ChatGPT chatbot.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:12]:
```suggestion
OpenAI is the AI research and deployment company behind ChatGPT. When you sign up with ChatGPT, it provides API access through an API key. After signing up at [https://platform.openai.com/playground](https://platform.openai.com/playground), you can create a personal account with limited access or a corporate account. You can use the playground to experiment with the API, too. A highlight of the API is that it accepts natural language commands similar to the ChatGPT chatbot. 
```

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md [hunk 16]
```diff
@@ -1,26 +1,26 @@
...
+## Enabling AI Analysis in Helm chart
 
-The `content-ai` configurations are available in the Helm Chart [values.yaml](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md) file as `contentAI`.
+The `content-ai` configurations are available in the Helm chart [values.yaml](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md) file as `contentAI`.
 
 The administrator can enable AI analysis for content in the `configuration` section of the ```core``` as shown in the following example.
 
 !!!note
-    When upgrading from older CF versions (for example, CF212) to CF213 or higher to enable AI analysis, the following configuration can be done during `helm upgrade`.
+    When you upgrade from older CF versions (for example, CF212) to CF213 or later to enable AI analysis, you can complete the configuration during during the `helm upgrade`.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:23]:
```suggestion
    When you upgrade from older CF versions (for example, CF212) to CF213 or later to enable AI analysis, you can complete the configuration during the `helm upgrade`.
```

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md [hunk 17]
```diff
@@ -132,14 +135,14 @@ configuration:
 
 For disbaling AI analysis for content, set ```enabled``` as ```false``` inside the contentAI section. Specifying the provider is not required to disable AI analysis.
 
-## Custom Configurations for AI Analysis
+## Custom configurations for AI analysis
 
-To customize AI analysis-related configurations, log in to WAS console for customizing any of the custom properties in the WCM Config Service (**Resource > Resource Environment > Resource Environment Providers > WCM_WCMConfigService > Custom Properties**).
+To customize AI analysis-related configurations, log in to the WAS <!-- Do you mean WebSphere Application Server here? Don't abbreviate other company's product names. --> console for customizing any of the custom properties in the WCM config service. Click **Resource > Resource Environment > Resource Environment Providers > WCM_WCMConfigService > Custom Properties**.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:140]:
```suggestion
To customize AI analysis-related configurations, log in to the WebSphere Application Server <!-- Do you mean WebSphere Application Server here? Don't abbreviate other company's product names. --> console for customizing any of the custom properties in the WCM config service. Click **Resource > Resource Environment > Resource Environment Providers > WCM_WCMConfigService > Custom Properties**.
```

**Review by biancaluna — COMMENTED** [review]
Hi @philip-menzies , please see comments and suggested changes.

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md (hunk 17 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:140]: OK

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md (hunk 16 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:23]: ok

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md (hunk 15 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:12]: ", too" here would be only for emphasis. It's not required grammatically.

File: docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md (hunk 14 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/container_deployment/wcm_content_ai_analysis.md:3]: if Web Content Management is a product, then the words can be initially capitalized. If the words are only abbreviated by WCM and not special in any other way they are not automatically proper nouns..

**Review by philip-menzies — COMMENTED** [review]
OK

**Pull request merged**